### ISSUE TITLE
Improve Feeds page

### ISSUE THREAD

**Issue by mlodic** [intelowlproject/GreedyBear#290]
Add:

- [ ] chance to look for IOCs
- [ ] chance to look for time ranges

**Comment by sristyanand00** [comment]
Hi @mlodic 
I’m interested in working on this issue and improving the Feeds page (IOCs, time ranges, and extracted payloads).  
If it’s okay, could you please assign this issue to me?

**Comment by sristyanand00** [comment]
Hi @mlodic,

I’ve completed work on the other issue you asked me to handle first and have raised a PR for it.

I’m now free to take this issue up as well. If it’s okay, I’d be happy to start working on it.

Thanks!

**Comment by ayushagarwalhere** [comment]
Hi @mlodic 👋
I'd love to work on this issue! I've set up GreedyBear locally and I'm familiar with the codebase. Could you please assign this to me?
Thank you!

**Comment by Riddhi0410** [comment]
Hi @mlodic 
I’m interested in working on this issue and improving the Feeds page. I've set up GreedyBear locally on my machine and I'm familiar with the codebase. Could you please assign this to me?
Thank you!

**Comment by rootp1** [comment]
@mlodic, @regulartim can I get this assigned this after #292

**Comment by cleo-coder** [comment]
Hi @mlodic 
 I'm a GSoC 2026 applicant who has submitted a proposal 
for the Dashboard Modularization project. I'd like to work on this 
issue as a contribution. Could you assign it to me?

**Comment by piyushzgautam99-ship-it** [comment]
Hi, I'd like to work on this. Can I be assigned?

**Comment by regulartim** [comment]
Hey @piyushzgautam99-ship-it ! This issue's description is quite vague and more a general idea then a concrete task. What exactly are you planning to do?

**Comment by piyushzgautam99-ship-it** [comment]
Hi @regulartim ,

I plan to improve the Feeds page by adding IOC detection/display, time range information, extracted payloads column, and InteOwl links for IP addresses. I'll first explore the codebase and break these into concrete tasks before implementation.

Thanks!

**Comment by regulartim** [comment]
The InteOwl links are already implemented and in production. I would suggest the following process: get familiar with the code base and pick one of the improvements you mentioned. If you found one, open a separate issue, where you describe the improvement you want to implement. We can discuss that particular feature there and I'll eventually assign you. Sounds good @piyushzgautam99-ship-it ? 

Edit: we currently don't have the underlying data to fill an "extracted payloads" column, to my knowledge.

**Comment by piyushzgautam99-ship-it** [comment]
@regulartim Sounds good, thanks! I'll explore the codebase and open a separate issue once I identify a suitable improvement.

**Comment by cclts** [comment]
@regulartim Hey I wanted to double-check the current status of the remaining items.

It seems that:

* "chance to look for IOCs" appears to have been implemented in #419.
* "chance to look for time ranges" appears to have backend support from #1103, and it seems the original issue may have been waiting for the frontend work on the Feeds page.
* "column to list extracted payloads" is the only one that blocked by payload extraction support at the time.

Am I understanding this correctly, or are there still gaps in the original IOC lookup / time-range requirements that haven't been addressed yet?

**Comment by regulartim** [comment]
Hey @cclts ! 
- "chance to look for IOCs": I am not 100% sure what @mlodic meant by that. But this issue concerns the frontend, so I don't think it was fixed in #419 .
- "chance to look for time ranges": Correct, we have the underlying data and an open PR #1338 for the API but no frontend integration. Although @rootp1 already built a frontend, IIRC.
- "column to list extracted payloads": @opbot-xd currently works on the payload extraction and it will be finished by the end of August, probably.

**Comment by mlodic** [comment]
yep my idea was having the chance to look for IOCs from the frontend

**Comment by regulartim** [comment]
But isn't that already implemented by the enrichment search bar?

**Comment by mlodic** [comment]
I forgot about that, sorry, yes indeed
